pkgs/top-level/all-packages.nix

```diff
@@ -15910,6 +15910,8 @@ with pkgs;
 
   thinkingRock = callPackage ../applications/misc/thinking-rock { };
 
+  nylas-mail = callPackage ../applications/networking/mailreaders/nylas-mail { };
```

Comment:
@7c6f434c Even though there's no other version? Wont that be confusing for users?